### ISSUE TITLE
About ground-truth of the 90 real training data

### ISSUE THREAD

**Issue by JiaxiongQ** [Vandermode/ERRNet#16]
Should the real training data be normalized to range [0,1] during training? I find your training code doesn't do that.

**Comment by Vandermode** [comment]
Simply dividing 255 is what you want.
